eastdetector/__init__.py

```diff
@@ -195,12 +183,24 @@ def detect(self, image: np.ndarray, params=TextDetectorParams()) -> Union[None,
         :return: An (N x 4 x 2) array, where N is the amount of boxes found and the (4 x 2) subarray holds the corner
                  points.
         """
-        if self.__closed:
-            raise RuntimeError("TextDetector: Session has already been closed.")
-        image_resized, (ratio_h, ratio_w) = _resize_image(image)
+        image_resized, (ratio_h, ratio_w) = _resize_image(image=image, max_side_len=640)
+        h, w, _ = image_resized.shape
+
+        net = cv2.dnn.readNet(self.pb_path)
+        # construct a blob from the image and then perform a forward pass of
+        # the model to obtain the two output layer sets
+        blob = cv2.dnn.blobFromImage(image_resized, 1.0, (w, h),
+                                     (123.68, 116.78, 103.94), swapRB=True, crop=False)
+        net.setInput(blob)
+
+        start = time.time()
+        (score_map_nchw, geo_map_nchw) = net.forward(self.layer_names)
+        end = time.time()
+        print("[INFO] session run text detection took {:.6f} seconds".format(end - start))
```

Comment:
please do not leave any prints inside of libraries. You can let the user decide what they want to print. If you find this really message really necessary you should add a "verbose: bool=false" flag to the params